eogrow/pipelines/download_batch.py

```diff
@@ -96,6 +96,9 @@ class Schema(Pipeline.Schema):
                 " method of `SentinelHubBatch` during the creation process."
             ),
         )
+        num_retries_on_partial: int = Field(
```

Comment:
Wondering if `num_retries` would be sufficient, since the description is also present. There are probably also not any other contexts where retries make sense for this pipeline

Reply:
i wanted to make it clear that retries are only done when the result is `PARTIAL`
but i guess that is also in the description